{"name": "test_logout", "status": "broken", "statusDetails": {"message": "NameError: name 'locators' is not defined", "trace": "self = <login_test.TestLogin object at 0x03C8A790>\n\n    def test_logout(self):\n    \n        try:\n            driver = self.driver\n            homepage = HomePage(driver)\n            # homepage.welcome_link_is_displayed()\n            time.sleep(5)\n>           homepage.click_welcome()\n\ntests\\login_test.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.HomePage.flows.HomePage object at 0x041165F8>\n\n    def click_welcome(self):\n>       self.driver.find_element(*locators.Page.welcome_link_id).click()\nE       NameError: name 'locators' is not defined\n\npages\\HomePage\\flows.py:28: NameError"}, "attachments": [{"name": "test_logout_05-07-2020_15-01-57", "source": "3d3fb433-29cc-41a5-ba0b-47561cc5e43d-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "4b506c05-837f-41d3-8316-0e8480392f0e-attachment.txt", "type": "text/plain"}], "start": 1593950512559, "stop": 1593950518281, "uuid": "a70d826a-4787-4ca7-b574-895d913250f7", "historyId": "606d9f046945fff19ea92d7b1a10f16b", "testCaseId": "e89a5a0e9aefe451ee262eba82370b01", "fullName": "tests.login_test.TestLogin#test_logout", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('test_setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-BFIIM5B"}, {"name": "thread", "value": "16004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.login_test"}]}